integration-tests/bridges/environments/polkadot-kusama/bridges_polkadot_kusama.sh

```diff
@@ -299,14 +266,40 @@ case "$1" in
           "ws://127.0.0.1:9942" \
           "//Alice" \
           1002 1000 4 524288
-      # set XCM version of remote AssetHubKusama
+      # governance set XCM version of remote AssetHubKusama on AHP
       force_xcm_version \
           "ws://127.0.0.1:9942" \
           "//Alice" \
           1000 \
           "ws://127.0.0.1:9910" \
           "$(jq --null-input '{ "parents": 2, "interior": { "X2": [ { "GlobalConsensus": "Kusama" }, { "Parachain": 1000 } ] } }')" \
           $XCM_VERSION
+      # governance set XCM version of remote BridgeHubKusama on BHP
+      force_xcm_version \
+          "ws://127.0.0.1:9942" \
+          "//Alice" \
+          1002 \
+          "ws://127.0.0.1:8943" \
+          "$(jq --null-input '{ "parents": 2, "interior": { "X2": [ { "GlobalConsensus": "Kusama" }, { "Parachain": 1002 } ] } }')" \
+          $XCM_VERSION
+      ;;
+  init-asset-hub-polkadot-local)
+      ensure_polkadot_js_api
+      # create pool for DOT and wKSM
+      create_pool \
+          "ws://127.0.0.1:9910" \
+          "//Bob" \
+          "$(jq --null-input '{ "parents": 1, "interior": "Here" }')" \
+          "$(jq --null-input '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": "Kusama" }] } }')"
+      # Create liquidity in the pool
+      add_liquidity \
+          "ws://127.0.0.1:9910" \
+          "//Bob" \
+          "$(jq --null-input '{ "parents": 1, "interior": "Here" }')" \
+          "$(jq --null-input '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": "Kusama"}] } }')" \
+          100000000000 \
+          250000000000 \
```

Comment:
pardon, my suggestion was wrong. this is leads to prettxy accurate pricing:
```suggestion
          100000000000 \
          4000000000000 \
```